content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+{{< warning >}}
```

Comment:
Please use the boilerplate for this warning instead.

Reply:
This warning is copied from the experimental feature boilderplate at content/en/boilerplates/experimental-feature-warning.md.

Reply:
What I mean is to use the boiletplate command:

{{< boilerplate "experimental-feature-warning" >}}

Reply:
Done, thanks!